cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -754,6 +889,21 @@ Replaced by:
 SHOW {POINT \| RANGE \| TEXT} INDEXES
 ----
 
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+USING BTREE INDEX
+----
+a|
+B-tree indexes are deprecated.
+
+Replaced by:
+[source, cypher, role="noheader"]
+----
+USING {POINT \| RANGE \| TEXT} INDEX
+----
```

Comment:
Was this cherry picked to `4.4` ???

Reply:
Yes, in https://github.com/neo4j/neo4j-documentation/pull/1558/files